package.json

```
@@ -46,11 +46,17 @@
     "lodash": "^4.17.15"
   },
   "resolutions": {
+    "eslint": "^8.25.0",
```

Even though we're on `8.25.0` I still think it makes sense to match what we're specifying elsewhere.

```suggestion
    "eslint": "^8.19.0",
```

---

The change in this file is reverted.